From 1358ebfcea4d88e06de08f77780905cb406a3d08 Mon Sep 17 00:00:00 2001
From: auto3000 <auto3000@users.noreply.github.com>
Date: Sun, 11 Jun 2017 19:36:59 +0200
Subject: [PATCH] build: disable strip of .so library in utils/

Non-stripped binaries and libraries are recommended by Yocto QA.
This modification disable object stripping and compiler optimizations.


---
 utils/Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/utils/Makefile b/utils/Makefile
index d9b389bc..c16da4d9 100644
--- a/utils/Makefile
+++ b/utils/Makefile
@@ -2,9 +2,9 @@
 CC  ?= gcc
 CXX ?= g++
 
-CFLAGS += -Wall -Wextra -std=c99 -fPIC -O3
-CXXFLAGS += -Wall -Wextra -Wshadow -std=gnu++0x -fPIC -O3
-LDFLAGS += -Wl,-O1 -Wl,--no-undefined -Wl,--strip-all
+CFLAGS += -Wall -Wextra -std=c99 -fPIC 
+CXXFLAGS += -Wall -Wextra -Wshadow -std=gnu++0x -fPIC 
+LDFLAGS += -Wl,--no-undefined 
 
 ifeq ($(shell pkg-config --atleast-version=0.22.0 lilv-0 && echo true), true)
 CXXFLAGS += -DHAVE_NEW_LILV
